ackhandler/sent_packet_handler.go

```diff
@@ -12,6 +12,18 @@ import (
 	"github.com/lucas-clemente/quic-go/utils"
 )
 
+const (
```

Comment:
I don't see a benefit, and they are not too many here. For constants that are only used in one place, I'd rather have them as local ad as close to their use as possible.